examples/stable_diffusion_v2/text_to_image.py

```diff
@@ -468,6 +475,23 @@ def main(args):
         default="logging.INFO",
         help="log level, options: logging.DEBUG, logging.INFO, logging.WARNING, logging.ERROR",
     )
+    parser.add_argument(
+        "--check_safety",
+        action="store_true",
+        help="set this flag to use a safety checker",
+    )
+    parser.add_argument(
+        "--clip_ckpt_path",
+        type=str,
+        default=None,
```

Comment:
ckip ckpt path is already uploaded to the download server. It's better support auto download if `clip_ckpt_path` is None.

Reply:
fixed